news/_posts/2020-06-15-spark-ai-summit-june-2020-agenda-posted.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: Spark+AI Summit (June 22-25th, 2020, San Francisco) agenda posted
```

Comment:
It's not really in San Francisco is it?

Reply:
Hehe good point. It's online / virtual.

Reply:
Yeah, I think calling that out would be good, so we can have more of the community participate. People may be used to skipping events outside of their local and it's nice that we can have a more inclusive event this time :)

Reply:
yea it's virtual, but AFAIK the time schedule of the talks still follows the SF timezone. Shall we mention that as well?

Reply:
But looks like the timezone isn't mentioned in the summit site either. I guess we can just simply follow what the site says.

Reply:
fixed